tests/test_geolocation_search.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+from pytest import FixtureRequest
+import requests
+import datetime
+from typing import Generator
+from bson.objectid import ObjectId
+from utils import MongoDb
+
+geolocation_host = "http://localhost:83"
+NewYork = {"name": "NewYork", "lat": 40.730610, "lng": -73.935242}
+JerseyCity = {"name": "JerseyCity", "lat": 40.719074, "lng": -74.050552}
```

Comment:
We're using the underscore naming convention in the project

Reply:
ok will change that